### ISSUE TITLE
drive does not appear in explorer

### ISSUE THREAD

**Issue by citizenserious** [maharmstone/btrfs#401]
Hit there,

I installed the latest release 1.7.8.1 successfully.
I also added the registry entry UpgradedSystem and set it to "1" as described.

Somehow nothing is happening when I plug my btrfs formatted drive in my windows computer, even after a restart.
The drive does not appear in the explorer, even tough it is visible (like before) in the Disk Management.

Did I overlook something?

Thank you very much.

**Comment by maharmstone** [comment]
Is the driver running? Try `sc query btrfs` from an admin command prompt. If not, what does `sc start btrfs` say?

**Comment by citizenserious** [comment]
It seems to be stopped, but I don't know why.

SERVICE_NAME: btrfs
        TYPE               : 1  KERNEL_DRIVER
        STATE              : 1  STOPPED
        WIN32_EXIT_CODE    : 31  (0x1f)
        SERVICE_EXIT_CODE  : 0  (0x0)
        CHECKPOINT         : 0x0
        WAIT_HINT          : 0x0

**Comment by ohhmm** [comment]
Yes, release must be signed to be installable securely https://docs.microsoft.com/en-us/windows-hardware/drivers/develop/signing-a-driver

**Comment by citizenserious** [comment]
> Yes, release must be signed to be installable securely https://docs.microsoft.com/en-us/windows-hardware/drivers/develop/signing-a-driver

do you know how to do this? 
The description here states "The driver is signed, so should work out of the box on modern versions of Windows."

**Comment by ohhmm** [comment]
The driver must be signed with publicly verifiable certificate.

**Comment by ohhmm** [comment]
That's on maintainer

**Comment by ohhmm** [comment]
@citizenserious, 
Anyway, you can buy new certificate and sign your build with your public certificate.

**Comment by maharmstone** [comment]
@citizenserious, the driver works out of the box as is, you don't need to mess around with signatures. Try installing the latest version as described in the README (i.e. not using pnputil).

**Comment by ohhmm** [comment]
Thanks @maharmstone,

Misleading was unsigned inf part. See how to sign inf too: https://learn.microsoft.com/en-us/windows-hardware/drivers/devtest/inf2cat?source=recommendations
![image](https://user-images.githubusercontent.com/190779/191088805-9e73df0d-1314-4d22-9d7b-2b9147242c58.png)

Looks like the driver however is signed.
![image](https://user-images.githubusercontent.com/190779/191088879-e2c696cb-dff0-4f83-875e-9d46db296804.png)

**Comment by maharmstone** [comment]
The inf is signed

**Comment by maharmstone** [comment]
Closing old issues
